Frameworks/UIKit/UIActionSheet.mm

```diff
@@ -110,10 +110,12 @@ - (void)_UIActionSheet_initInternal:(const FrameworkElement&)xamlElement {
         objcwinrt::callback([weakSelf] (const Controls::ContentDialog&, const Controls::ContentDialogOpenedEventArgs&) {
             __strong UIActionSheet* strongSelf = weakSelf;
 
-            strongSelf->_isContentDialogVisible = YES;
+            if (strongSelf != nil) {
```

Comment:
> != nil [](start = 26, length = 7)

nit: Much prefer a simple boolean comparison for pointers: 'if (strongSelf)' ...